app/Models/WpOrg/Asset.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Models\WpOrg;
+
+use App\Enums\AssetType;
+use App\Models\BaseModel;
+use Database\Factories\WpOrg\AssetFactory;
+use Illuminate\Database\Eloquent\Concerns\HasUuids;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class Asset extends BaseModel
+{
+    use HasUuids;
+
+    /** @use HasFactory<AssetFactory> */
+    use HasFactory;
+
+    protected $fillable = [
```

Comment:
We also want a discriminator for the source, let's just call it `repository` and set its value to `"wp_org"`.  There should also be a created timestamp.